### PR TITLE
fix calculating timeout of sudo command

### PR DESCRIPTION
**Pull request by greg-latuszek**

### CODE REVIEW

File: test/cmd/unix/test_cmd_sudo.py [hunk 1]
```diff
@@ -62,6 +63,37 @@ def test_failing_with_timeout(buffer_connection, command_output_and_expected_res
...
+    buffer_connection.remote_inject_response([command_output])
+    cmd_pwd = Pwd(connection=buffer_connection.moler_connection)
+    cmd_pwd.timeout = 0.2
+    cmd_sudo = Sudo(connection=buffer_connection.moler_connection, password="pass", cmd_object=cmd_pwd)
+    cmd_sudo.terminating_timeout = 0  # no additional timeout for Ctrl-C..till..prompt (shutdown after cmd timeout)
+    start_time = time.time()
+    with pytest.raises(CommandTimeout):
+        cmd_sudo()
+    duration = time.time() - start_time
+    assert duration > 0.2
```
Comment (marcin-usielski) [inline test/cmd/unix/test_cmd_sudo.py]: `duration >= 0.2` ? It means no less than 0.2s ?

Comment (greg-latuszek) [reply test/cmd/unix/test_cmd_sudo.py]: you are right - should be `duration >= 0.2` - you can fix it

File: test/cmd/unix/test_cmd_sudo.py [hunk 2]
```diff
@@ -62,6 +63,37 @@ def test_failing_with_timeout(buffer_connection, command_output_and_expected_res
...
+    buffer_connection.remote_inject_response([command_output])
+    cmd_pwd = Pwd(connection=buffer_connection.moler_connection)
+    cmd_pwd.timeout = 0.8
+    cmd_sudo = Sudo(connection=buffer_connection.moler_connection, password="pass", cmd_object=cmd_pwd)
+    cmd_sudo.terminating_timeout = 0  # no additional sudo timeout for Ctrl-C..till..prompt (shutdown after cmd timeout)
+    start_time = time.time()
+    with pytest.raises(CommandTimeout):
+        cmd_sudo(timeout=0.5)
+    duration = time.time() - start_time
+    assert duration > 0.5
```
Comment (greg-latuszek) [inline test/cmd/unix/test_cmd_sudo.py]: should be `duration >= 0.5`

Comment (marcin-usielski) [inline test/cmd/unix/test_cmd_sudo.py]: Maybe `duration >= 0.5` ?

Comment (greg-latuszek) [reply test/cmd/unix/test_cmd_sudo.py]: right

**Pull request merged**